InvenTree/templates/InvenTree/settings/plugin.html

```diff
@@ -35,8 +35,6 @@
         <h4>{% trans "Plugins" %}</h4>
         {% include "spacer.html" %}
         <div class='btn-group' role='group'>
-            {% url 'admin:plugin_pluginconfig_changelist' as url %}
-            {% include "admin_button.html" with url=url %}
```

Comment:
Question: Why is this button removed?

Reply:
I have worked out a way to get this back in now :)